src/OpenLoco/src/Vehicles/VehicleHead.cpp

```diff
@@ -874,7 +874,9 @@ namespace OpenLoco::Vehicles
     bool VehicleHead::updateLand()
     {
         Vehicle2* vehType2 = _vehicleUpdate_2;
-        if ((!(vehType2->var_73 & Flags73::isBrokenDown) || (vehType2->var_73 & Flags73::isStillPowered)) && status == Status::approaching)
+
+        // If dont have any running issue and is aproaching
+        if ((!vehType2->has73Flags(Flags73::isBrokenDown & Flags73::isStillPowered)) && status == Status::approaching)
```

Comment:
should be `!vehType2->has73Flags(Flags73::isBrokenDown) || vehType2->has73Flags(Flags73::isStillPowered)`

Reply:
Ah, sure